src/useRestActions.ts

```diff
@@ -29,73 +29,75 @@ const useRestActions = <Schema>(conf?: Partial<Config>, initialData?: Schema[])
 
   // index
   const indexConfig = mergeConfig(config.index, config)
-  const handleSuccessIndex = (result: Schema[]) => {
+  const handleSuccessIndex = useCallback((result: Schema[]) => {
     const { createSetItems } = actionCreators
     dispatch(createSetItems(result))
-  }
+  }, [ actionCreators, dispatch ])
   const index = useCreateAction<Schema>(handleSuccessIndex as HandleSuccess<Schema>, shouldMakeRequest(indexConfig) ? "GET" : undefined, indexConfig, actionCreators, dispatch)
-  const indexStable = useCallback(index, [])
```

Comment:
I think we're fooling the linter here.
We're making a 'stable' version of index, in which the linter is not able to see which dependencies it has.

Dependencies are:
`handleSuccessIndex, indexConfig, actionCreators, dispatch`

`handleSuccessIndex` should be a useCallback itself, with its own dependencies. (`actionCreators` and `dispatch`)